src/hooks/useTasks.js

```diff
@@ -3,61 +3,69 @@ import useNow from './useNow'
 import { useTasksSubscription } from './subscription-hooks'
 import { getAdjudicationPhase } from '../utils/dispute-utils'
 import * as DisputesTypes from '../types/dispute-status-types'
-
 import { useCourtConfig } from '../providers/CourtConfig'
 
 export default function useTasks() {
   const courtConfig = useCourtConfig()
-  const { tasks, error } = useTasksSubscription()
+  const { tasks, fetching, error } = useTasksSubscription()
   const now = useNow()
 
-  return { openTasks: useOpenTasks(tasks, now, courtConfig), error }
+  return { openTasks: useOpenTasks(tasks, now, courtConfig), fetching, error }
```

Comment:
Could we declare the variable outside ? Like:
```suggestion
  const openTasks = useOpenTasks(tasks, now, courtConfig)
  return { openTasks, fetching, error }
```